README.md

```diff
@@ -65,10 +65,10 @@ To test authorization for Management Service locally, make the following changes
 To use the XP Go modules as API dependencies, `replace` directives need to be used when consuming both the Management and Treatment API modules since the API modules uses local relative path.
 
 ```go
-replace github.com/gojek/xp/clients => github.com/gojek/turing-experiments/clients v0.0.0
-replace github.com/gojek/xp/common => github.com/gojek/turing-experiments/common v0.0.0
-replace github.com/gojek/xp/management-service => github.com/gojek/turing-experiments/management-service v0.0.0
-replace github.com/gojek/xp/treatment-service => github.com/gojek/turing-experiments/treatment-service v0.0.0
+replace github.com/caraml-dev/xp/clients => github.com/caraml-dev/xp/clients v0.0.0
```

Comment:
These replace directives are incorrect.

If we're keeping the use of `gojek/xp` references when using the Go modules as API dependencies, then the following should be the right directives:
```
replace github.com/gojek/xp/clients => github.com/gojek/turing-experiments/clients v0.0.0
...

Considering the shift to `caraml-dev` organisation is a permanent one, all internal repositories that reference Github XP could have their dependencies changed to point directly to `caraml-dev/xp` as well.
```

Reply:
> If we're keeping the use of gojek/xp references when using the Go modules as API dependencies
I think we're changing the name of the Go modules from `gojek/xp` to `caraml-dev/xp`, or at least that's what we're doing for Turing too.

> Considering the shift to caraml-dev organisation is a permanent one, all internal repositories that reference Github XP could have their dependencies changed to point directly to caraml-dev/xp as well.
Yep, I'll update those once this has been merged :)

Reply:
I've just removed the replace directives from the README since they aren't useful anymore (the Go module paths are now named exactly as their URLs)